README.md

```diff
@@ -2,15 +2,15 @@
 
 ## Roster
 
-| Name     | Job                | Country of Origin     | Japan Inc. Score |
-| -------- | ------------------ | --------------------- | ---------------- |
-| Traviss  | Front-end Engineer | Washington State, USA | Sep. 2006        |
-| Eli      | Sr. Engineer (FE)  | Oregon, USA           | Oct. 2007        |
-| Daniel   | Developer (FS)     | Germany               | ?                |
-| Kaz      | Sales Consultant   | Japan                 | 1975             |
-| Chris    | --                 | Canada                | ?                |
-| Luke     | Front-End Lead     | Leeds, UK             | Feb. 2016        |
-| David    | --                 | France                | ?                |
-| Mary     | Front-end Engineer | Minnesota, USA        | ?                |
-| Anderson | --                 | Brazil/Japan?         | ?                |
-| Ruben    | Engineer (BE)      | Netherlands           | Mar. 2016        |
+| Name     | Job                | Country of Origin     | Japan Inc. Score.   |
+| -------- | ------------------ | --------------------- | ------------------- |
+| Traviss  | Front-end Engineer | Washington State, USA | Sep. 2006           |
+| Eli      | Sr. Engineer (FE)  | Oregon, USA           | Oct. 2007           |
+| Daniel   | Developer (FS)     | Germany               | ?                   |
+| Kaz      | Sales Consultant   | Japan                 | 1975                |
+| Chris    | --                 | Canada                | ?                   |
+| Luke     | Front-End Lead     | Leeds, UK             | Feb. 2016           |
+| David    | --                 | France                | ?                   |
+| Mary     | Front-end Engineer | Minnesota, USA        | ?                   |
+| Anderson | Technical Lead     | Japan/Brazil/Japan    | Oct. 1992/July 2017 |
```

Comment:
Can you choose a date. I eventually want to use this calculate time in Japan. Going to be tough for your situation though :)